game-core/src/main/java/games/strategy/engine/framework/startup/ui/PlayerTypes.java

```diff
@@ -18,6 +17,8 @@
 @Value
 public class PlayerTypes {
 
+  public static final String DOES_NOTHING_PLAYER = "Does Nothing (AI)";
```

Comment:
This is now just a label for the DNAI? What do you think of adding a `LABEL` suffix to help clarify that?